docker-compose.yml

```diff
@@ -1,35 +1,36 @@
-# Run from source root by following command:
-#   docker-compose -f cloudify-manager/docker-compose.yml up
-#
-# Directory has to contain:
-#  - cloudify-common
-#  - cloudify-agent
-#  - cloudify-manager
-#  - config.yaml (configuration for manager)
-#  - cloudify_internal_ca_cert.pem
-#  - cloudify_internal_ca_key.pem
-
 version: "3.8"
 
 services:
-  manager:
-    container_name: manager
-    image: cloudify-manager-aio
-    volumes:
-      - ../config.yaml:/etc/cloudify/config.yaml
-      - ../cloudify_internal_ca_cert.pem:/tmp/cloudify_internal_ca_cert.pem
-      - ../cloudify_internal_ca_key.pem:/tmp/cloudify_internal_ca_key.pem
+#  mgmtworker:
+#    container_name: mgmtworker
+#    build: mgmtworker
+#    env_file:
+#      - .env
+#    volumes:
```

Comment:
I would like to stay `mgmtworker` as a "workable", so I decided to mount types and cert for now.

Reply:
Actually it's entirely commented-out.  Maybe you could just drop these lines (for clarity if nothing else)

Reply:
Probably we will run that service in the future, so I decided to mount required volumes for that.